source/services/index.md

```diff
@@ -0,0 +1,105 @@
+An `Ember.Service` is a long-lived `Ember.Object` available application-wide. Services are singletons, and only one instance of a given service can exist at a time.
+
+To define a service, extend the `Ember.Service` base class:
+
+```app/services/shopping-cart.js
+export default Ember.Service.extend({
+});
+```
+
+Like any `Ember.Object`, a service can have properties and methods of its own.
+
+```app/services/shopping-cart.js
+export default Ember.Service.extend({
+  items: [],
+
+  add(item) {
+    this.get('items').pushObject(item);
+  },
+
+  remove(item) {
+    this.get('items').removeObject(item);
+  },
+
+  empty() {
+    this.get('items').setObjects([]);
+  }
+});
+```
+
+### Accessing services
+
+To access a service, inject it with `Ember.inject`:
+
+```app/components/cart-contents.js
+export default Ember.Component.extend({
+  shoppingCart: Ember.inject.service()
```

Comment:
Can we teach the maximal form first `shoppingCart: Ember.inject.service('shopping-cart')` before showing the shorthand?